app/scripts/views/mixins/password-mixin.js

```diff
@@ -44,12 +37,8 @@ define(function (require, exports, module) {
           this.logViewEvent('password.visible');
         } else {
           passwordField.attr('type', 'password');
-          if (this.isPasswordAutoCompleteDisabled()) {
-            passwordField.attr('autocomplete', 'off');
-          } else {
-            passwordField.removeAttr('autocomplete')
-              .removeAttr('autocorrect').removeAttr('autocapitalize');
-          }
+          passwordField.removeAttr('autocomplete')      
+               .removeAttr('autocorrect').removeAttr('autocapitalize');
```

Comment:
I think you can probably reduce this to a single line and just do something like this:

```js
passwordField.removeAttr('autocomplete autocorrect autocapitalize');
```

---

> #### [`.removeAttr( attributeName )`](https://api.jquery.com/removeAttr/#removeAttr-attributeName)
> _An attribute to remove; as of version 1.7, it can be a space-separated list of attributes._